_posts/2025-06-30-pyos-scipy-2025.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: single
+title: "pyOpenSci at SciPy 2025 - science, Python and Community"
```

Comment:
The title of the post is the h1 (heading 1) and will show up at the top of the page. 

the excerpt (below) is used for the blog "cards" on the website and search engines often use it as well when they list pages.